### DISCUSSION TITLE
How to export & import "Translations"

### DISCUSSION THREAD

**Discussion by chantlong** [directus/directus#14811] [Archive]
Let's say I have a collection called `A` that has a translation field. (This creates a collection `A_translations` and `Languages`
Within the `A_translation` collection data model, I added `type` and value`.

When I export collection `A` I get a CSV like the following.

<img width="454" alt="Screen Shot 2022-08-03 at 17 37 19" src="https://user-images.githubusercontent.com/17346712/182563633-b02c4fb2-8786-4b6e-b478-e32fe9b936e5.png">

But when I create a new collection `B` with the same fields, and try to reimport it, you see the Translations become numbers in an array. When I click on the record, the value for `type` and `value` show up empty.

<img width="696" alt="Screen Shot 2022-08-03 at 17 39 03" src="https://user-images.githubusercontent.com/17346712/182564052-23782d81-5425-4baf-ac99-89b72e917ba4.png">

Do you how I can make translations import correctly?

**Comment by rijkvanzanten** [comment]
> When I export collection A I get a CSV like the following.

How did you export collection A?

**Reply by chantlong** [reply]
<img width="1000" alt="Screen Shot 2022-08-04 at 9 52 42" src="https://user-images.githubusercontent.com/17346712/182739994-921d2b6b-139e-4505-a22b-fd7d404a9d0b.png">
Here is the A Collection. I added the `Translation` field to be displayed

I went to Collection A, clicked Export Items as CSV and chose the following fields to export.
<img width="822" alt="Screen Shot 2022-08-04 at 9 50 23" src="https://user-images.githubusercontent.com/17346712/182740049-93492b1c-faa1-46c3-9824-688f720da8bd.png">

I am reimporting Collection A from Microsoft SQL to Postgres though. Not sure if that matters.

**Reply by rijkvanzanten** [reply]
Ah, make sure to also export the translations language field, otherwise the import can't know what language the imported value has

**Reply by chantlong** [reply]
Thank you will try that!
